From: Arnaud Ferraris <arnaud.ferraris@collabora.com>
Date: Fri, 3 Apr 2020 17:13:55 +0200
Subject: arm64: dts: allwinner: pinephone: change backlight brightness levels

Improve the backlight brightness values so we have a wider usable range.

Signed-off-by: Arnaud Ferraris <arnaud.ferraris@collabora.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts | 10 +++++-----
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts | 13 ++++++-------
 2 files changed, 11 insertions(+), 12 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts
index 098a358b..b162a34 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.1.dts
@@ -38,11 +38,11 @@ &backlight {
 	 * value here was chosen as a safe default.
 	 */
 	brightness-levels = <
-		774  793  814  842
-		882  935  1003 1088
-		1192 1316 1462 1633
-		1830 2054 2309 2596
-		2916 3271 3664 4096>;
+		392  413  436  468
+		512  571  647  742
+		857  995  1159 1349
+		1568 1819 2103 2423
+		2779 3176 3614 4096>;
 	num-interpolated-steps = <50>;
 	default-brightness-level = <400>;
 };
diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
index 14f8273..d4e2f8d 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
@@ -33,14 +33,13 @@ &backlight {
 	 * chosen as a safe default.
 	 */
 	brightness-levels = <
-		5000 5248 5506 5858 6345
-		6987 7805 8823 10062 11543
-		13287 15317 17654 20319 23336
-		26724 30505 34702 39335 44427
-		50000
-	>;
+		392  413  436  468
+		512  571  647  742
+		857  995  1159 1349
+		1568 1819 2103 2423
+		2779 3176 3614 4096>;
 	num-interpolated-steps = <50>;
-	default-brightness-level = <500>;
+	default-brightness-level = <400>;
 };
 
 &lis3mdl {
